supervisor/core.py

```diff
@@ -97,15 +97,6 @@ async def connect(self):
         # Check supervisor version/update
         if self.sys_dev:
             self.sys_config.version = self.sys_supervisor.version
-        elif (
```

Comment:
Why was this removed?

Reply:
Is not needed anymore with the new build logic